app/model/ContentSubscriptionPlanOps.scala

```diff
@@ -56,7 +56,8 @@ object ContentSubscriptionPlanOps {
     }
 
     def availableForCheckout: Boolean = in.charges.billingPeriod.isRecurring || in.charges.billingPeriod == SixWeeks
-    def availableForRenewal: Boolean = in.charges.billingPeriod == Quarter || in.charges.billingPeriod == OneYear || in.charges.billingPeriod == Year
+
+    def availableForRenewal: Boolean = in.charges.billingPeriod == Quarter || in.charges.billingPeriod == OneYear || in.charges.billingPeriod == Year || in.charges.billingPeriod == ThreeMonths
```

Comment:
Consider `List(Quarter, OneYear, Year, ThreeMonths) contains in.charges.billingPeriod`?